data/successor 1 introduction.txt

```diff
@@ -565,12 +565,12 @@ mission "Successors: First Contact 3"
 					goto betterlatethannever
 			action
 				set "Successors FC3: captured kijra-pet"
-			`	You rummage around in the other cargo containers until you find a bit of spare fabric scraps and leaf litter, a couple of handfuls of which you add to the animal's cage. With that sorted, you make your preparations for takeoff.`
+			`	You rummage around in the other cargo containers until you find a bit of spare fabric scraps and leaf litter, a couple of handfuls of which you add to the animal's cage. With that sorted out, you make your preparations for takeoff.`
 				goto end
 			label takeoff
 			action
 				set "Successors FC3: captured kijra-pet"
-			`	With the animal sorted, you make your preparations for takeoff.`
+			`	With the animal not having too many problems, you make your preparations for takeoff.`
```

Comment:
```suggestion
			`	With the animal stowed away, you make your preparations for takeoff.`
```
Your suggestion is a little wordy for me, but I agree that something more than "sorted" is needed. Does this phrasing work for you?